demo/site/preBuild/src/createRedirects.ts

```diff
@@ -47,9 +47,23 @@ const createApiRedirects = async (): Promise<Redirect[]> => {
     for (const redirect of response.redirects) {
         let source: string | undefined;
         let destination: string | undefined;
+        let has: Redirect["has"];
 
         if (redirect.sourceType === "path") {
-            source = redirect.source;
+            source = redirect.source.replace(/:/g, "\\:"); // escape colon, otherwise it is used for next.js regex path matching  (https://nextjs.org/docs/pages/api-reference/next-config-js/redirects#regex-path-matching)
+
+            // query parameters have to be defined with has, see: https://nextjs.org/docs/pages/api-reference/next-config-js/redirects#header-cookie-and-query-matching
+            if (source?.includes("?")) {
+                const searchParams = new URLSearchParams(source.split("?")[1]);
```

Comment:
Could this lead to unintended behavior if the URL contains multiple `?`? (https://stackoverflow.com/a/2924187)

Reply:
yes thank you.

I've adapted the code in the last commit to:
redirect.source.split("?").slice(1).join("?");

this should take everything after the first "?" and now worked with this url:
`/redirect?path=test3&someValue=t?1234`